components/HeroNotificationBox.tsx

```diff
@@ -0,0 +1,28 @@
+import Arrow from "../public/arrow_forward.svg";
+import Image from "next/image";
+import { FC } from "react";
+
+type HeroNotificationBoxProps = {
+  title: string;
+  content: string;
```

Comment:
```suggestion
  title?: string;
  content?: string;
```

these props are not required right now